frontend/src/components/AddDebuggingSessionModal.vue

```diff
@@ -34,20 +42,39 @@ const missingValues = computed(() => {
 const emit = defineEmits(['createDebuggingSession'])
 
 async function createNewDebuggingSession() {
-    loading.value = true;
-    try {
-        const newDebuggingSession = await debuggingSessionsStore.createDebuggingSession({
-            datasetId: selectedDataset.value!.id,
-            modelId: selectedModel.value!.id,
-            name: sessionName.value,
-            sample: true
-        });
-
-        closeDialog();
-        emit('createDebuggingSession', newDebuggingSession);
-    } finally {
-        loading.value = false;
+  const mlWorkers = await api.getMLWorkerSettings();
+  const isWorkerAvailable = mlWorkers.find(
+    value => {
+      if (project.value?.mlWorkerType === 'EXTERNAL') {
```

Comment:
For me this logic should be in the backend

Reply:
Mapped to https://linear.app/giskard/issue/GSK-1099/disable-actions-that-require-external-worker-if-no-worker-is-connected

Reply:
Mapped to https://linear.app/giskard/issue/GSK-1099/disable-actions-that-require-external-worker-if-no-worker-is-connected